reporting-app/app/helpers/dashboard_helper.rb

```diff
@@ -2,6 +2,8 @@
 
 module DashboardHelper
   def determine_dashboard_view
+    return "exemption_approved" if @member_status&.status == MemberStatus::EXEMPT
```

Comment:
I don't think this is working I dont see where MemberStatus::EXEMPT gets set in the code anywhere, I also do see a ExemptionDeterminationService that is called but I don't think its working , I set my age as low and it does not seem to be working